static/panes/editor.js

```diff
@@ -456,6 +460,12 @@ Editor.prototype.initEditorActions = function () {
             this.tryPanesLinkLine(ed.getPosition().lineNumber, true);
         }, this)
     });
+
+    // this.editor.addAction({
```

Comment:
You marked this as resolved: but there's still a block of commented-out code here? Can you remove please, we try not to leave confusing unused code snippets commented out.